src/app/build/build.component.html

```diff
@@ -59,17 +74,20 @@ <h4>
 
     <ul class="build-details" ng-hide="jobs.length == '0'" id="build-details">
       <li class="parent">
-          <em ng-show="build.provider == 'github'" class="ng-binding ng-hide"><i class="icon logo-github"></i> brigade-cli</em> 
-          <!-- Pending API hookup:
+        <em ng-show="build.provider == 'github'" class="ng-binding ng-hide">
+          <i class="icon logo-github"></i> {{ build?.provider }}</em>
+        <!-- Pending API hookup:
           <em ng-show="build.provider == 'dockerhub'" class="ng-binding ng-hide"><img src="assets/images/docker.png" alt="dockerhub"> brigade-cli</em>
-          <em ng-show="build.provider == 'slackbot'" class="ng-binding ng-hide"><img src="assets/images/slack.png" alt="Slack"> brigade-cli</em> 
+          <em ng-show="build.provider == 'slackbot'" class="ng-binding ng-hide"><img src="assets/images/slack.png" alt="Slack"> brigade-cli</em>
           <em ng-show="!(
-            build.provider === 'slackbot' || 
-            build.provider === 'dockerhub' || 
+            build.provider === 'slackbot' ||
+            build.provider === 'dockerhub' ||
             build.provider === 'github'
           )" class="ng-binding"> brigade-cli</em>  -->
-          : <em class="ng-binding">exec</em> 
-          started a build via <em class="ng-binding">commit #master</em>
+        :
+        <em class="ng-binding">{{ build?.type }}</em>
+        started a build via
+        <em class="ng-binding">commit {{ revision?.commit }}</em>
       </li>
 
       <!-- Pending API hookup:
```

Comment:
This is hard-coded so far, right? 
We need to update the services (both the mock and the real ones) to get this data - do we have all Brigade API methods in place to plug this in?